niap-cc/Permissions/Tester/app/build.gradle

```diff
@@ -15,52 +15,102 @@
  */
 
 apply plugin: 'com.android.application'
-
+def homePath = System.properties['user.home']
+def applicationName = 'Tester'
 android {
     signingConfigs {
-        debug {
+        bundle{
+            storeFile file("../security/rsa2048/rsa-2048.jks")
+            storePassword "android"
+            keyPassword "android"
+            keyAlias "platform"
         }
-        release {
+        platform {
```

Comment:
I think we should either remove this and make it a TODO: for partners, or use something like the rsa-2048.jks (along with a TODO message about updating it with their own key) so partners can use their own platform signing key for the device under test.

Reply:
This setting helps to build the build-variants in a simple command set like

gradlew assembleNormal gradlew assembleNoperm... 

It need to assign a key to these each variant so we should not simply remove it.
As you know we hadn't prepare a simple way to build these variants automaticaly

How about changing jks file names to more common, 
and then add todo comment and share instruction of those settings?

Reply:
This and the instructions from the other commit sound good to me; thanks!